src/Common/src/Interop/Unix/libcrypto/Interop.X509.cs

```diff
@@ -29,6 +29,9 @@ internal static partial class libcrypto
         internal static extern SafeX509Handle X509_dup(SafeX509Handle handle);
 
         [DllImport(Libraries.LibCrypto)]
+        internal static extern SafeX509NameHandle X509_NAME_dup(IntPtr handle);
```

Comment:
We're not going to directly p/invoke into any non-Windows functions in the future.  While no one has started shimming libssl yet, libcrypto is underway, and we shouldn't be adding to debt now that we're in-progress on that library.